Source/ERF.H

```diff
@@ -51,97 +51,127 @@ grow_box_by_one(const amrex::Box& b)
   return amrex::grow(b, 1);
 }
 
-//
-// AmrLevel-derived class for hyperbolic conservation equations for stellar
-// media
-//
+struct SolverChoice {
```

Comment:
@pkjha-aero is this going to eventually deprecate `ProbParm`?

Reply:
The variables we define in `ProbParm` in `prob_par.H` are specific to the problem and are utilized in `Exec/*` and not in `Source/*`. But `SolverChoice` is meant for the those things that apply to problems in general and are utilized in `Source/*`